data.sql

```diff
@@ -14,4 +14,34 @@ INSERT INTO public.animals(
 	
 	INSERT INTO public.animals(
 	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-	VALUES (4, 'Devimon', '2017-05-12', 5, 'true', 11);
\ No newline at end of file
+	VALUES (4, 'Devimon', '2017-05-12', 5, 'true', 11);
+
+	-- Project part 2
+
+INSERT INTO public.animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (5, 'Charmander', '2020-02-08', 0, 'false', 11);
+
+	INSERT INTO public.animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (6, 'Plantmon', '2021-11-15', 2, 'true', 5.7);
+
+	INSERT INTO public.animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (7, 'Squirtle', '1993-04-02', 3, 'false', 12.13);
+
+	INSERT INTO public.animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (8, 'Angemon', '2005-05-12', 1, 'true', 45);
```

Comment:
- [ ] Kindly you need to pay more attention to the project requirements, the fourth first INSERT must have a negative  weight_kg value. let's correct this 💯 👍🏻